fastlane/docs/Jenkins.md

```diff
@@ -68,3 +68,23 @@ To show the **generated screenhots** right in `Jenkins`
 Save and run. The result should look like this:
 
 ![JenkinsIntegration](../assets/JenkinsIntegration.png)
+
+## Change log
+
+You might need to automatically generate a changelog from your commits, particularly if you deploy your apps to an internal system.
+
+Jenkins might be already passing relevant information to your environment so that fastlane can generate the change log using an appropriate action.
+
+Here's an example integration if you use the Jenkins Git plugin and the `changelog_from_git_commits` fastlane action:
+
+```ruby
+  changelog = changelog_from_git_commits(
+    between: [ENV['GIT_PREVIOUS_SUCCESSFUL_COMMIT'], ENV['GIT_COMMIT']],
+    pretty: "* (%h) [%aN] %s"
+  )
+  changelog = "Empty changelog..." if changelog.to_s.length == 0
+```
+
+And this is how it would look like if that `changelog` was passed to the `hockeyapp` action:
+
+![Jenkins ChangeLog to HockeyApp](../assets/jenkins_hockeyapp_changelog.png)
```

Comment:
Thanks for adding the screenshot! It looks good, however, for the sake of presenting only the most relevant information, could you crop out just the "Release Notes" part of your screenshot please? Like this: 
![jenkinschangelog](https://cloud.githubusercontent.com/assets/279407/21407428/2f1ce3b4-c785-11e6-8267-329497421333.png)